src/organisation/containers/organisation/organisation.component.spec.ts

```diff
@@ -174,4 +189,61 @@ describe('OrganisationComponent', () => {
     expect(store.pipe).toHaveBeenCalled();
     expect(component.getOrganisationDetails()).toEqual(mockOrganisationDetails);
   });
+
+  it('should get the User Details from the Store, and set it on orgData.', () => {
+
+    component.getOrganisationDetailsFromStore();
+
+    expect(store.pipe).toHaveBeenCalled();
+    expect(component.getOrganisationDetails()).toEqual(mockOrganisationDetails);
+  });
+
+  describe('showChangePbaNumberLink property', () => {
+    const CHANGE_LINK = By.css('#change-pba-account-numbers__link');
+
+    it('should show the \'Change\' link when false', () => {
+      component.showChangePbaNumberLink = true;
+      fixture.detectChanges();
+
+      const changeLink = fixture.debugElement.query(CHANGE_LINK);
+
+      expect(changeLink).toBeTruthy();
+    });
+
+    it('should not show the \'Change\' link when false', () => {
```

Comment:
backticks would look better here to remove the need to put \'

Reply:
changed